src/mbgl/renderer/tile_pyramid.cpp

```diff
@@ -290,16 +265,19 @@ void TilePyramid::handleWrapJump(float lng) {
 
     if (wrapDelta) {
         std::map<OverscaledTileID, std::unique_ptr<Tile>> newTiles;
+        std::map<UnwrappedTileID, std::reference_wrapper<Tile>> newRenderTiles;
         for (auto& tile : tiles) {
             auto newID = tile.second->id.unwrapTo(tile.second->id.wrap + wrapDelta);
             tile.second->id = newID;
             newTiles.emplace(newID, std::move(tile.second));
         }
         tiles = std::move(newTiles);
 
-        for (auto& renderTile : renderTiles) {
-            renderTile.id = renderTile.id.unwrapTo(renderTile.id.wrap + wrapDelta);
+        for (auto& tile : renderTiles) {
+            UnwrappedTileID newID = tile.first.unwrapTo(tile.first.wrap + wrapDelta);
+            newRenderTiles.emplace(newID, tile.second);
```

Comment:
Could this be handled without reallocation (e.g. turn flags on/off)?

Reply:
here we've got to update map keys. Unfortunately, `std::map` does not have reserve API to be used for the new map instance. On the other hand, we do not have hundreds of visible tiles at a frame, so it should not be a problem.

Reply:
Cleared in offline discussion. size of render tiles is around 10. Previous code already uses map (renderTiles) and here we are just std::move to it.